### PR TITLE
Odst survey

### PR DESCRIPTION
**Pull request by Mikehickey7**
Still need to connect to Jira Story with this branch

-Created form folder 
     -verification of fields filled in before submit
     -successfully queries surveys to connect or create survey
     -successfully connects or creates unique questions
TODO: need to submit answers still along with surveryResponse

### CODE REVIEW

**Review by ericcater — CHANGES_REQUESTED** [review]

**Comment by yurisim** [comment]
image

File: apps/ods-api/schema.prisma [hunk 1]
```diff
@@ -66,12 +66,13 @@ model Survey {
...
   orgs            Org[]
   questions       Question[]
+  questionsHash   String?           @unique
   surveyResponses SurveyResponse[]
 }
 
 model Question {
   id      String   @id @default(uuid())
-  prompt  String
+  prompt  String   @unique
```
Comment (yurisim) [inline apps/ods-api/schema.prisma:75]: Need to change the other seed scripts if you are going to do this.

**Comment by yurisim** [comment]
image
 idk if you guys are trolling but if stuff like this leaks  into production it makes us look bad :/

**Comment by yurisim** [comment]
image

Try to have the route be somewhat related to the module in naming

**Comment by yurisim** [comment]
Did you guys check to see what this looks like @ different screen sizes before submitting?

image

**Comment by yurisim** [comment]
image

Instead of calling it all over the place, wrap the entire thing in 1 div and make the child elements fit to size. Don't duplicate CSS like this

**Comment by yurisim** [comment]
image

Thought we said submit button on top?

**Comment by yurisim** [comment]
Be consistent

image

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -26,7 +26,7 @@ jobs:
       - run: yarn install --frozen-lockfile --prefer-offline
       - run: yarn audit
       - name: Generate Prisma clients
-        run: yarn nx run-many --target=prisma-generate --all=true --parallel=false
+        run: yarn nx run-many --target=prisma-generate --all
```
Comment (yurisim) [inline .github/workflows/ci.yml:29]: Revert these changes

Comment (sayntonge) [reply .github/workflows/ci.yml:29]: no u

File: apps/ods-api/src/prisma/singleton.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { mockDeep, mockReset, DeepMockProxy } from 'jest-mock-extended';
```
Comment (yurisim) [inline apps/ods-api/src/prisma/singleton.ts:1]: Delete this file

**Pull request closed without merging**